students/EricAdams/timing_context_manager/timer_context_manager_write_to_file_obj.py

```diff
@@ -0,0 +1,34 @@
+#! /usr/bin/env python
+'''
+Create a simple timer context manager to take a file object and write to it
+'''
+import time
+import sys
+
+
+class Timer():
+
+    def __init__(self, file_object=sys.stdout):
+        self.file_object = file_object
+
+    def __enter__(self):
+        self.start = time.clock()
+        self.name = 'timer_writing_to_file'
```

Comment:
it'd be nice to have a way for the user to give it their own name...

But this all looks good otherwise.